first_task_search_engine/prepare_dataset.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import pandas as pd
+from polyglot.detect import Detector
+
+
+def get_lang_info(text: str):
+    """
+    File contains questions in different languages.
+    We get only English texts.
+    """
+    try:
+        lang = Detector(text, quiet=True).language
+    except Exception:
+        lang = np.nan
+    return lang
+
+
+def run_prepare_dataset():
```

Comment:
General advice: it's usually better to use pure functions (that do not interfere with the outer world as disk, memory, etc.).
In this case - it's better to get df object, modify it and return df object.
If you want I can talk about function purity and it's advantages in the next lecture (just remind me)

Reply:
Ok, I will ask you on the next lecture